### DISCUSSION TITLE
Never sends

### DISCUSSION THREAD

**Discussion by sharkmullet** [Extended-Thunder/send-later#358] [Q&A] [answered]
I haven't changed any settings, but it stopped working about 4 days ago.

My email goes into the drafts, but it never sends it. It sits in the draft box with the correct time in the "send later' column, but it sits there past that time and never goes out.

Is it user error?

Thunderbird 78.12.0 (32-bit)
send later version 8.9.5

**Comment by jph-sendlater** [comment, marked as answer]
Try following the troubleshooting steps described in the docs:
https://extended-thunder.github.io/send-later/#troubleshooting

Let me know if you're still experiencing problems after that.

**Reply by sharkmullet** [reply]
Thanks, I tried steps 1-5 and they all didn't produce errors.

The message goes into my drafts folder with the correct time, but when the time comes it doesn't send.

**Reply by sharkmullet** [reply]
just want to follow up on this.  It would be nice if I could schedule an email today for Monday

**Reply by jph-sendlater** [reply]
What does the 'Send Later' item in the status bar indicate (bottom right of the main Thunderbird window)? It should say something like `Send Later [PEND 1]` if there is a message waiting to send, or `Send Later [IDLE]` if there are no scheduled messages.

You might want to try repairing the drafts folder: https://extended-thunder.github.io/send-later/#drafts-cleanup

If that doesn't help, then you'll need to take a debug log in order to start tracking down what might be happening. Follow the steps in: https://extended-thunder.github.io/send-later/#debug-logging
And record a log through the process of scheduling a message, and waiting for it to become due to send (give it a couple minutes after it's due to make sure Send Later has had a chance to pick it up).

**Reply by sharkmullet** [reply]
It says Idle but then goes to PEND 1 when I hit send on an email.

I'll take a look at repairing drafts

**Reply by sharkmullet** [reply]
it's working now.  Maybe it was the repair of the drafts folder.  I have 40 or so drafts, not sure if it could be related to that

**Reply by jph-sendlater** [reply]
Glad to hear it's working now!
